src/main/java/frc/robot/commands/AutoChargeStationBalance.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.commands;
+
+import com.ctre.phoenix.sensors.WPI_Pigeon2;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Calibrations;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+public class AutoChargeStationBalance extends CommandBase{
+    private final DriveSubsystem drive;
+    private final WPI_Pigeon2 gyro;
+
+    public AutoChargeStationBalance(DriveSubsystem drive){
+        this.drive = drive;
+        this.gyro = drive.getGyro();
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {}
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
```

Comment:
Can we add a check to see if we're at the end of autonomous and if so, set velocity to 0?